### ISSUE TITLE
do away with import ordering restrictions

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar-angular#371]
The [FullCalendar Angular docs](https://fullcalendar.io/docs/angular) require that the `@fullcalendar/angular` package go before any other fullcalendar-related package. Otherwise you'll get a JS error like `Please import the top-level fullcalendar lib before attempting to import a plugin.` Example configuration:

```ts
import { FullCalendarModule } from '@fullcalendar/angular'; // the main connector. must go first
import dayGridPlugin from '@fullcalendar/daygrid'; // a plugin
import interactionPlugin from '@fullcalendar/interaction'; // a plugin
```

Also, the `registerPlugins` exists for a similar reason: to prevent the `.component.ts` file from importing its own plugin files, which might violate the ordering. Example:

```ts
FullCalendarModule.registerPlugins([ // register FullCalendar plugins
  dayGridPlugin,
  interactionPlugin
]);
```

It would be great to do away with this ordering restriction as well as `registerPlugins`. For an explanation of why this ordering restriction exists in the first place, see https://github.com/fullcalendar/fullcalendar/issues/6371

**Comment by Dorus** [comment]
When VSCode Organize imports breaks your build because `bootstrap5Plugin` goes before `Calendar` alphabetically...

**Comment by arshaw** [comment]
This is fixed in [v6.0.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.0.0)
